notebooks/landing-page.md

```diff
@@ -1,3 +1,65 @@
-# Cookbook Overview
+# (Replace_with_your_title) Cookbook
 
-This is the the landing page for your cookbook - update as neccessary!
+This Project Pythia Cookbook covers ... (replace `...` with the main subject of your cookbook ... e.g., *working with radar data in Python*)
+
+## Motivation
+
+(Add a few sentences stating why this cookbook will be useful. What skills will you, "the chef", gain once you have reached the end of the cookbook?)
+
+## Structure
+(State one or more sections that will comprise the notebook. E.g., *This cookbook is broken up into two main sections - "Foundations" and "Example Workflows."* Then, describe each section below.)
+
+### Section 1 ( Replace with the title of this section, e.g. "Foundations" )
+(Add content for this section, e.g., "The foundational content includes ... ")
+
+### Section 2 ( Replace with the title of this section, e.g. "Example workflows" )
+(Add content for this section, e.g., "Example workflows include ... ")
+
+## Running the Notebooks
+You can either run the notebook using [Binder](https://mybinder.org/) or on your local machine.
+
+### Running on Binder
+
+The simplest way to interact with a Jupyter Notebook is through
+[Binder](https://mybinder.org/), which enables the execution of a
+Jupyter Book in the cloud. The details of how this works are not
```

Comment:
Can you add a link to the JupyterBook project?

Reply:
Yep, thanks!